packages/tests/src/account.ts

```diff
@@ -121,6 +127,13 @@ export class Account {
         return result;
     }
 
+    setMinExpectedLt(minExpectedLt: string) {
+        if (this.cachedBocLt && minExpectedLt > this.cachedBocLt) {
```

Comment:
You perform ordered comparison of the string representations of the numbers. There may be an unexpected comparison result.